client/settings/payment-methods-selector/index.js

```diff
@@ -16,12 +16,12 @@ import PaymentMethodCheckboxes from '../../components/payment-methods-checkboxes
 import PaymentMethodCheckbox from '../../components/payment-methods-checkboxes/payment-method-checkbox';
 import './style.scss';
 
+const availablePaymentMethods = [
+	'woocommerce_payments_giropay',
+	'woocommerce_payments_sofort',
+	'woocommerce_payments_sepa',
+];
 const PaymentMethodsSelector = ( { className } ) => {
```

Comment:
is this change related to this PR?

Reply:
Yes - the `availablePaymentMethods` is defined within the body of `PaymentMethodsSelector`.
Which means that its reference changes every time `PaymentMethodsSelector` runs. But `availablePaymentMethods` is used inside a hook. And since its reference changes at each render, and since it needs to be declared as a hook dependency, the hook would not be memoized properly.
By extracting it to a `const` outside of the body of `PaymentMethodsSelector`, its reference doesn't change.